CHANGELOG.md

```diff
@@ -4,13 +4,18 @@
 
 ### 💥 Breaking Changes
 
-* `DataViewConfig` no longer directly supports the passing of `GridConfig` parameters.
-  Instead, all direct `GridConfig` toggles must be passed through the new `gridOptions`
-  parameter.
+* Apps should update their Typescript dependency to v5.1. This should be a drop-in for most
+  applications, or require only minor changes. Note that Hoist has not yet adopted the updated
+  approach to decorators added in TS v5, maintaining compatibility with the "legacy" syntax.
```

Comment:
@lbwexler does this look like a useful / correct description of the TS upgrade from an app's POV?

Reply:
👍🏻